app/models/reports/result.rb

```diff
@@ -184,6 +185,8 @@ def calculate_period_info(range)
         bp_control_range = period.blood_pressure_control_range
         hsh[period] = {
           name: period.to_s,
+          start_date: period.start_date.strftime(DATE_FORMAT),
+          end_date: period.end_date.strftime(DATE_FORMAT),
```

Comment:
This will always be the same as `bp_control_end_date`, right? it makes sense to have it as a separate thing, just confirming.

Reply:
Yep, the ltfu graph shows patients recorded until _this_ month, so I had to add a separate `start_date` and `end_date` to show in the labels and description.